.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 cache: yarn
 matrix:
   include:
-  - node_js: '6'
```

Comment:
You'll also want to update the requirements section of the readme https://github.com/maxmind/GeoIP2-node#requirements